app/components/planning_applications/immunity_details_assessment_component.html.erb

```diff
@@ -57,26 +57,22 @@
             <strong>Relevant dates: </strong> <%= evidence_group.date_range %>
           </p>
 
-          <% if evidence_group.applicant_comment.present? %>
-            <div class="govuk-inset-text" style="background-color: #f3f2f1; border-left: 10px solid #b1b4b6;">
-              <p class="govuk-body">
-                <strong>Applicant comment</strong>
-              </p>
-              <p class="govuk-body">
-                <%= evidence_group.applicant_comment %>
-              </p>
-            </div>
+          <% if applicant_comment = evidence_group.applicant_comment %>
+            <p class="govuk-body govuk-!-margin-top-2">
+              <%= bops_ticket_panel(colour: "grey") do |ticket| %>
+                <% ticket.with_body { applicant_comment } %>
+                <% ticket.with_footer { "By applicant" } %>
+              <% end %>
```

Comment:
I'd put the component outside the `<p>` tag so that you can adjust the bottom margin